tests/common/common.go

```diff
@@ -18,7 +18,7 @@ package common
 
 import "flag"
 
-const upImg = "ghcr.io/rancher/elemental-toolkit/elemental-green:v1.3.0"
+const upImg = "ghcr.io/rancher/elemental-toolkit/elemental-green:v0.10.7-g3e4a3c56"
```

Comment:
Why go back to such an old release?

Reply:
To keep grub compatibility, this is compatible with v1.1.0 but not really with v1.2 or v1.3. In fact I think we should stick with v1.1.0 here, but not sure we ever published it to the registry. This was one of the things I wanted to review after merging this.

The issue is with the `${mode}` and `${img}` variables in grub, in v1.2 and v1.3 we assume the image is referred by the ${mode} and the dracut module computes the path. This is hard make it compatible with v1.1.0, v1.2.0 and the upcoming v2.0.x as this implies having several paths adjustments.

If we assume the ${img} and is the full path and ${mode} is just to discriminate recovery vs active args this turns to be easier in terms of backwards compatibility. We have to keep in mind in the grub domain further changes are around the corner with the btrfs snapshotter and that our backward compatibility constraint should stick to v1.1.0 version, as being the latest released elemental-toolkit version. Because of that I took a very conservative approach, because we are likely to need revising it in any case.